test/common/tests.js

```diff
@@ -1315,9 +1315,21 @@ module.exports = (sql, driver) => {
       })
     },
 
+    'force ODBC driver verify' (cfg) {
+      assert.ok(cfg.conn_str.startsWith('Driver=ODBC Driver 17 for SQL Server'))
+    },
+
+    'force ODBC driver query' (done) {
+      sql.query('select serverproperty(N\'servername\') as servername; ', (err, result) => {
+        assert.ok(result.recordset[0].servername)
+        done(err)
+      })
+    },
+
     'login failed' (done, message) {
       const config = readConfig()
       config.user = '__notexistinguser__'
+      config.options.trustedConnection = false
```

Comment:
I'm not sure this should be added - the connections will use self-signed or non-encrypted connections so we have to trust the connection?